openapi/multi-yaml/paths/payment_method_groups.yaml

```diff
@@ -0,0 +1,48 @@
+post:
+  summary: Create a Payment Method Group
+  description: |
+    You can create payment methods groups ahead of time, then associate payment methods and easily filter them.
+  operationId: CreatePaymentMethodGroup
+  tags:
+    - Payment Method Groups
+  parameters:
+    - $ref: ../components/parameters/authorization-header.yaml
+    - $ref: ../components/parameters/sub-account.yaml
+  responses:
+    '201':
+      description: Payment method group was created successfully
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: ../components/schemas/Envelope.yaml
+              - properties:
+                  type:
+                    example: payment_method_group
+                  data:
+                    $ref: ../components/schemas/PaymentMethodGroupResponse.yaml
+
+get:
+  summary: List a Payment Method Groups
```

Comment:
Should this be `List Payment Method Groups` instead?

Reply:
Yea, good catch!